docs/migrations.md

```diff
@@ -194,25 +196,57 @@ class Migration(migrations.Migration):
     ]
 ```
 
+`field_name` is the name of the StreamField on the Page model that contains the blocks to migrate. 
+`block_type` is the type of block within a StreamField that contains the data to be migrated.
+
+StreamBlocks can themselves also contain child blocks. 
+`migrate_page_types_and_fields` will run the given mapper function on 
+all StreamBlocks within the StreamField and 
+all child blocks that are of `block_type`.
+
+The `data` that gets passed to the `forward_mapper` or `backward_mapper`
+is a JSON-compatible Python `dict` that corresponds to the block's schema.
```

Comment:
Is it already a `dict`? Do we need the `data = dict(data)` lines in the example mappers? I feel like I've not needed those in the past. Will try to confirm with my migration in progress.

Reply:
I was wondering that myself when writing this. I don't think we need that line? But I want to make sure.

Reply:
Nope, looks like the casting as a dict is unnecessary.